contracts/PangolinExchange.sol

```diff
@@ -89,6 +90,49 @@ contract PangolinExchange is Ownable, IAssetsExchange, ReentrancyGuardUpgradeabl
     return true;
   }
 
+
+  /**
+   * Adds supported asset
+   * @dev _asset asset to be added
+  **/
+  function setAsset(bytes32 _asset, address _address) external override onlyOwner {
```

Comment:
maybe we can accept an array with assets?

Reply:
This could be a good idea